backend/fms_core/viewsets/sample.py

```diff
@@ -285,224 +284,20 @@ def update(self, request, *args, **kwargs):
-        derived_sample_values_queryset = (
-            DerivedSample.objects
-            .filter(id__in=derived_sample_ids)
-            .values(
-                "id",
-                "biosample_id",
-                "biosample__alias",
-                "sample_kind_id",
-                "tissue_source_id",
-                "biosample__collection_site",
-                "experimental_group",
-                "biosample__individual_id",
-            )
-        )
-        derived_samples = { ds["id"]: ds for ds in derived_sample_values_queryset }
-
-        serialized_data = []
-        for sample in samples.values():
-            derived_sample = derived_samples[sample["first_derived_sample"]]
-            is_pool = sample["derived_count"] > 1
-            is_library = sample["is_library"]
-            data = {
-                'id': sample["id"],
-                'biosample_id': derived_sample["biosample_id"] if not is_pool else None,
-                'name': sample["name"],
-                'alias': derived_sample["biosample__alias"] if not is_pool else None,
-                'volume': sample["volume"],
-                'depleted': sample["depleted"],
-                'concentration': sample["concentration"],
-                'child_of': sample["child_of"],
-                'extracted_from': sample["extracted_from"] if not is_pool else None,
-                'individual': derived_sample["biosample__individual_id"] if not is_pool else None,
-                'container': sample["container_id"],
-                'coordinates': sample["coordinates"],
-                'sample_kind': derived_sample["sample_kind_id"] if not is_pool or (is_pool and not is_library) else "POOL",
-                'is_library': is_library,
-                'is_pool': is_pool,
-                'projects': sample["derived_samples__project"],
-                'process_measurements': sample["process_measurement"],
-                'tissue_source': derived_sample["tissue_source_id"] if not is_pool else None,
-                'creation_date': sample["creation_date"],
-                'collection_site': derived_sample["biosample__collection_site"] if not is_pool else None,
-                'experimental_group': json.dumps(derived_sample["experimental_group"]) if not is_pool else None,
-                'quality_flag': sample["quality_flag"],
-                'quantity_flag': sample["quantity_flag"],
-                'created_by': sample["created_by"],
-                'created_at': sample["created_at"],
-                'updated_by': sample["updated_by"],
-                'updated_at': sample["updated_at"],
-                'deleted': sample["deleted"],
-                'comment': sample["comment"],
-            }
-            serialized_data.append(data)
-
+        serialized_data = fetch_sample_data([], samples_queryset, self.request.query_params)
         return Response(serialized_data)
```

Comment:
change this line to: 

`return Response({
            'results': serialized_data,
            'count': samples_queryset.count()
  })
`

Change a few lines in the SampleListComponent in the frontend i.e. the render method of sample's project in the column list to :

`
render: (_, sample) => (sample.projects &&
        <Link to={`/projects/${sample.projects}`}> {projectsByID[sample.projects]?.name} </Link>
      ),
`


And the list will show. There's a a few details that might have to change to be able to show all the fields.